plugin.video.dailymotion_com/default.py

```diff
@@ -487,7 +484,8 @@ def getUrl2(url):
         ff = "off"
     print 'The url is ::',url
```

Comment:
```suggestion
     print('The url is ::%s' % url)
```

Reply:
I will replace all the print statements with xbmc.log and resubmit
Those print statements are from eden days!